docs/detections/detection-engine-intro.asciidoc

```diff
@@ -86,7 +86,7 @@ Indicator match rules provide a powerful capability to search your security data
 
 In addition, the following support restrictions are in place:
 
-* {elastic-sec} does not support the use of either cold or frozen {ref}/data-tiers.html[tier data] with indicator match rules.
+* {elastic-sec} does not support the use of either cold or frozen {ref}/data-tiers.html[tier data] with indicator match rules. However, the rule will search cold and frozen data tiers if they exist. To prevent this, configure the `excludedDataTiersForRuleExecution` <<exclude-cold-frozen-data-rule-executions,advanced setting>> (which applies to all rules in a space), or add a <<exclude-cold-frozen-data-individual-rules,Query DSL filter>> to individual rules. 
```

Comment:
```suggestion
* Indicator match rules don't support cold or frozen data. However, if cold or frozen  {ref}/data-tiers.html[data tiers] exist, indicator match rules still query them and return results, which are used when the rule executes. This can impact performance and even extend execution time. To ensure rules don't include cold and frozen data when executing, configure the `excludedDataTiersForRuleExecution` <<exclude-cold-frozen-data-rule-executions,advanced setting>> (which applies to all rules in a space), or add a <<exclude-cold-frozen-data-individual-rules,Query DSL filter>> to individual rules. 
```